src/Cache/Results.php

```diff
@@ -115,6 +115,11 @@ public function get_query_results_from_cache_cb( $result, Request $request ) {
 			return $result;
 		}
 
+		// For mutation, do not cache
+		if ( 'Mutation' === $request->get_query_analyzer()->get_root_operation() ) {
```

Comment:
We might want to consider changing this to `if ( 'Query' !== $request->get_query_analyzer()->get_root_operation() )`

Since WPGraphQL supports 3 operation types: 

- Query
- Mutation
- Subscription

While we don't ship with Subscriptions, it's possible to add them (I have plugin code that does it already!) and we shouldn't be caching those as well.

Let's be defensive here and bail for any operation other than Query.